messaging/rabbitmq_binding.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+
```

Comment:
I didn't see any module using MIT. I used the GPL but I guess MIT would also be fine.